packages/vue/config/storybook/home/home.js

```diff
@@ -16,32 +16,32 @@ storiesOf(" |Introduction", module)
     <div class="intro">
       <h1>StorefrontUI - Customization-first component library</h1>
       <p>
-        Welcome in our storybook!
+        Welcome to our storybook!
         You can use it to browse and play with StorefrontUI components.
-        On the left panel you will find every component that is currently available in the library. 
+        On the left panel, you will find every component that is currently available in the library. 
         You can play with props and CSS modifiers of each component from <b>Knobs menu</b>. 
-        Components marked as <i>WIP</i> will much likely change and they;re not safe to use.
-        On the <b>notes</b> menu you will find additional information about component such as available SCSS variables.
+        Components marked as <i>WIP</i> will much likely change and they're not safe to use.
+        On the <b>notes</b> menu, you will find additional information about component such as available SCSS variables.
       </p>
       <p>
         <b>StorefrontUI is currently in ALPHA, which means it is not suitable for production! Expect things to be broken and APIs to possibly change.</b>
       </p>
       <h2> Join us! </h2>
-        <p> StorefrontUI is a completely new approach for component libraries oriented around customization possibilities instead of out of the box possibilities. It's a great goal but it will take a lot of time to achieve it only by ourselves. The project is open source and free to use. If you'd like to support us and join the team please write to me: <b> filip [at] vuestorefront.io </b>;)</p>
+        <p> StorefrontUI is a completely new approach for component libraries oriented around customization possibilities instead of out of the box possibilities. It's a great goal but it will take a lot of time to achieve it only by ourselves. The project is open source and free to use. If you'd like to support us and join the team, please write to me: <b> filip [at] vuestorefront.io </b>;)</p>
```

Comment:
`StorefrontUI is a completely new approach for component libraries. It is oriented around customisation possibilities instead of out of the box possibilities.`

Reply:
`It's a great goal, but it will take a lot of time to achieve it only by ourselves.`